requirements-dev.txt

```diff
@@ -9,3 +9,4 @@ pyyaml
 tox>=3.8
 tox-pip-extensions
 unidiff
+responses
```

Comment:
Maybe put after `pyyaml` to keep the requirements alphabetical.

Reply:
Adjusted.